From 317191848a64006249de30fb3ed388a88ff22a90 Mon Sep 17 00:00:00 2001
From: Will Woods <wwoods@redhat.com>
Date: Wed, 21 Mar 2012 19:47:46 -0400
Subject: [PATCH] shutdown: fix PATH

Some systems might not use /bin:/sbin in their paths anymore, but we're
still using it in initramfs, so make sure our PATH is correct.
---
 modules.d/99shutdown/shutdown.sh |    1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/99shutdown/shutdown.sh b/modules.d/99shutdown/shutdown.sh
index c8ca664..8817a4c 100755
--- a/modules.d/99shutdown/shutdown.sh
+++ b/modules.d/99shutdown/shutdown.sh
@@ -10,6 +10,7 @@
 #!/bin/sh
 . /lib/dracut-lib.sh
 export TERM=linux
+PATH=/usr/sbin:/usr/bin:/sbin:/bin
 
 emergency_shell()
 {
